srcpkgs/python3-Flask-HTTPAuth/template

```diff
@@ -1,18 +1,18 @@
 # Template file for 'python3-Flask-HTTPAuth'
 pkgname=python3-Flask-HTTPAuth
-version=4.4.0
-revision=2
+version=4.5.0
+revision=1
 wrksrc=Flask-HTTPAuth-${version}
 build_style=python3-module
 hostmakedepends="python3-setuptools"
 depends="python3-Flask"
-checkdepends="${depends}"
 short_desc="Basic, Digest and Token HTTP authentication for Flask routes"
-maintainer="mobinmob <mobinmob@disroot.org>"
+maintainer="Orphaned <orphan@voidlinux.org>"
 license="MIT"
 homepage="https://github.com/miguelgrinberg/Flask-HTTPAuth"
 distfiles="${PYPI_SITE}/f/flask-httpauth/Flask-HTTPAuth-${version}.tar.gz"
-checksum=bcaaa7a35a3cba0b2eafd4f113b3016bf70eb78087456d96484c3c18928b813a
+checksum=395040fda2854df800d15e84bc4a81a5f32f1d4a5e91eee554936f36f330aa29
+make_check=no # needs packages not in repo
```

Comment:
Could you list at least one of them so if they are ever packaged, a simple grep through srcpkgs will show which tests can be enabled?

Reply:
Done.